README.mkd

```diff
@@ -0,0 +1,50 @@
+CCEXTRACTOR
+
+
+WHAT DO WE DO?
+
+Our work is to make subtitles for your favorite shows. Our goal is to make this facility accessible for all. Our software has many tools that would make your work easy to use. Our code is purely written in C. For more information click here.
+
+OUR PATICIPATION IN GOOGLE CODE-IN.
```

Comment:
Why are the headers in caps lock? Why not use GH markdown to achieve the proper header markup?